milli/src/search/new/resolve_query_graph.rs

```diff
@@ -19,47 +19,58 @@ pub struct PhraseDocIdsCache {
 }
 impl<'ctx> SearchContext<'ctx> {
     /// Get the document ids associated with the given phrase
-    pub fn get_phrase_docids(&mut self, phrase: Interned<Phrase>) -> Result<&RoaringBitmap> {
+    pub fn get_phrase_docids(
+        &mut self,
+        universe: Option<&RoaringBitmap>,
```

Comment:
This may lead to cache issues, so better remove this parameter for now

Reply:
I removed the parameter, thank you!